diff --git a/Makefile b/Makefile
index ae45ecc..7299fc7 100644
--- a/Makefile
+++ b/Makefile
@@ -1,10 +1,10 @@
 # --- COMPILER ----------------------------------------
-CC = mpiicc 
+CC = mpicc 
 
 # --- CFLAGS -----------------------------------------
-CFLAGS_gnu = -std=gnu99 -Wall -pedantic -O3 -ffast-math -msse4.2 -fopenmp 
+CFLAGS_gnu = -std=gnu99 -Wall -pedantic -O2 -mavx2 -mfma -mtune=haswell -march=haswell -fopenmp 
 CFLAGS_intel = -std=gnu99 -Wall -pedantic -O3  -xHOST -qopenmp 
-CFLAGS = $(CFLAGS_intel)
+CFLAGS = ${CFLAGS_gnu}
 
 # --- DO NOT CHANGE -----------------------------------
 CPP = cpp
